run_viz_scala.sh

```diff
@@ -11,21 +11,31 @@ SF=target/scala-2.11/spark3d_2.11-${VERSION}.jar
 HP=lib/jhealpix.jar,lib/plotly-assembly-0.3.0-SNAPSHOT.jar
 
 MASTERURL=yarn
-inputfn="LSST1Y/out_srcs_s1_0.fits"
+#inputfn="LSST1Y/out_srcs_s1_0.fits"
+ #inputfn=hdfs://134.158.75.222:8020//lsst/LSST1Y/out_srcs_s1_3.fits
+#spark-submit \
+   # --master ${MASTERURL} \
+   # --driver-memory 4g --executor-memory 28g --executor-cores 17 --total-executor-cores 102 \
+   # --jars ${SF},${HP} --packages ${PACK} \
+   # --class com.astrolabsoftware.spark3d.examples.VisualizePart \
+    #target/scala-${SCAlA_VERSION}/spark3d_${SCAlA_VERSION}-${VERSION}.jar \
+   # $inputfn "Z_COSMO,RA,DEC" true "onion" 8 0.0001 "username" "api_key"
 
-spark-submit \
-    --master ${MASTERURL} \
-    --driver-memory 4g --executor-memory 28g --executor-cores 17 --total-executor-cores 102 \
-    --jars ${SF},${HP} --packages ${PACK} \
-    --class com.astrolabsoftware.spark3d.examples.VisualizePart \
-    target/scala-${SCAlA_VERSION}/spark3d_${SCAlA_VERSION}-${VERSION}.jar \
-    $inputfn "Z_COSMO,RA,DEC" true "onion" 8 0.0001 "username" "api_key"
+ #inputfn=hdfs://134.158.75.222:8020/user/julien.peloton/dc2
+ #spark-submit \
+  #   --master ${MASTERURL} \
+  #  --driver-memory 4g --executor-memory 28g --executor-cores 17 --total-executor-cores 102 \
+  #  --jars ${SP},${HP} --packages ${PACK} \
+  # --class com.astrolabsoftware.spark3d.examples.VisualizePart \
+  #  target/scala-${SCAlA_VERSION}/spark3d_${SCAlA_VERSION}-${VERSION}.jar \
+   #  $inputfn "position_x_mock,position_y_mock,position_z_mock" false "octree" 512 0.001 "abualia4" "GFz0cUDumcKcnhpMd8qw"
```

Comment:
Again, the general rule is to never push sensitive information in a repo! Could you remove your username and api key?